src/sign_in.ts

```diff
@@ -1,12 +1,12 @@
 // Copyright 2023 the Deno authors. All rights reserved. MIT license.
 import { type OAuth2Client, setCookie } from "../deps.ts";
 import {
-  COOKIE_BASE,
-  getCookieName,
-  isSecure,
-  OAUTH_COOKIE_NAME,
-  redirect,
-  setOAuthSession,
+    COOKIE_BASE,
+    getCookieName,
+    isSecure,
+    OAUTH_COOKIE_NAME,
+    redirect,
+    setOAuthSession,
```

Comment:
Can you please revert these stylistic changes? You may be using a formatter different from [Deno's in-built one](https://deno.land/manual/tools/formatter). Once reverted, you can just run `deno fmt`.

Reply:
ahh didn't even spot that, yes I think it's been reverted now